geoportal-connectors/geoportal-harvester-waf/pom.xml

```diff
@@ -21,6 +21,11 @@
             <artifactId>geoportal-commons-constants</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
commons-text is already a dependency in a top level pom. It's ok to upgrade to 1.6